draft-ietf-scitt-architecture.md

```diff
@@ -908,6 +917,45 @@ To indicate that the content is an scitt configuration represented as JSON:
+
+The term "subject" provides an identifier of the issuer's choosing to refer to a given statement, and ensure that all associated claims can be attributed to the identifier chosen by the issuer.
+
+In simpler language, a statement could be some vendor specific software bill of materials, or results from a model checker, or static analyzer, where the issuer might identify themselves using via the `iss` claim, and the specific software that was analyzed via the `sub` claim.
+
+In {{RFC7523}}, the Authorization Server (AS) verifies Private Key JWT client authentication requests, and issues access tokens to clients configured to use "urn:ietf:params:oauth:client-assertion-type:jwt-bearer". This means the AS initially acts as a Verifier, and then later as an Issuer.
+
+In {{-rats-arch}} and {{-draft-ietf-rats-eat}}, when conveying Digital Letters of Approval (DLOAs), these claims are typically issued by a RATS Verifier, not a RATS attester, whereas Software Manifests (manifests) are issued by the software manufacturer.
+
+Section 6.3.6 and Section 6.3.7 of {{-draft-ietf-rats-eat}} comment on the use of COSE profiles, and uses the term "attester" to refer to the party which chooses a supported signature or encryption algorithm to secure claims (the act of issuance), and uses the term "verifier" to refer to the party which supports all allowed algorithms, and performs the decryption or verification operation (the act of verifying).
+
+Section 1.3.1 of {{-draft-ietf-rats-eat}} describes the relationship between Attesters which "issue Evidence", Verifiers which verify Evidence and "produce Attestation Results" which are consumed by Relying Parties.
+The term "verifier policy" is conceptually similar to the term "registration policy".
+
+{{FIPS.201}} defines "assertion" as "A verifiable statement from an IdP to an RP that contains information about an end user.".
+
+{{NIST.SP.800-63-3}} defines "assertion" as "A statement from a verifier to an RP that contains information about a subscriber.
+Assertions may also contain verified attributes."
+
+This document uses the term statement to refer to potentially unsecured data and associated claims, and Signed Statement and Receipt to refer to assertions from an issuer, or the transparency service.
+
+{{NIST.SP.1800-19}} defines "attestation" as "The process of providing a digital signature for a set of measurements securely stored in hardware, and then having the requester validate the signature and the set of measurements."
+
+NIST guidance "Software Supply Chain Security Guidance EO 14028" uses the definition from {{ISO.17000.2020}}, which states that an attestation is "The issue of a statement, based on a decision, that fulfillment of specified requirements has been demonstrated."
```

Comment:
And ISO 17000 Attestation is the basis for NIST Attestation (https://www.nist.gov/itl/executive-order-improving-nations-cybersecurity/software-supply-chain-security-guidance-2), which, e.g., conflicts with IETF RFC9334 Evidence or TCG Attestation. There be dragons.

Reply:
Lets identify them, and explain their conflict, thats the purpose of this section.